app/routes/missions/agile.md

```diff
@@ -0,0 +1,55 @@
+---
+meta:
+  title: GCN - AGILE
+---
+
+# Astrorivelatore Gamma a Immagini Leggero (AGILE)
+
+<img 
+  src="/_static/img/agile-logo.png"
+  width="200"
+  align="right"
+  alt="AGILE logo"
+/>
+
+**Launch Date:** April 23, 2007
+
+**Extended Mission Lifetime:** 2025+ (Pending ESA Review)
+
+**End of Operations:** No specific requirement (no consumables)
+
+**Data Archive:**
+https://agile.asdc.asi.it/
+
+[AGILE](http://agile.rm.iasf.cnr.it/) is a gamma-ray satellite from the Italian Space Agency with participation from INFN, INAF, and CIFS. It observes the high energy sky with two instruments and a calorimeter.
+
+| Instruments                       | Energy Range    | Field of View |
+| --------------------------------- | --------------- | ------------- |
+| SuperAGILE (SA)                   | 18 keV–60 keV   | &gt; 1 ster   |
+| Gamma Ray Imaging Detector (GRID) | 30 MeV–50 GeV   | &gt; 2.5 ster |
+| [Mini-Calorimeter (MCAL)          | 350 keV–100 MeV |               |
+
+**GCN Notice Types in GCN Classic and GCN Classic Over Kafka:**
+[Detailed Descriptions and Examples](https://gcn.gsfc.nasa.gov/agile.html)
+
+| Type      | Contents                               | Latency        |
+| --------- | -------------------------------------- | -------------- |
+| `WAKEUP`  | On-board processing                    | ~30 minutes    |
+| `REFINED` | Ground processing                      | 1–2 hours      |
+| `Refined` | Humans in the loop                     | ~2–5 hours     |
+| `MCAL`    | Ground processing of the MCAL triggers | 40–130 minutes |
```

Comment:
Use notice type names from https://github.com/lpsinger/pygcn/blob/main/gcn/notice_types.py. Same for all other files.